azext_edge/edge/_help.py

```diff
@@ -1560,8 +1558,6 @@ def load_iotops_help():
             --private-key-file "newopc.pem"
             --public-key-secret-name public-secret-name
             --private-key-secret-name private-secret-name
-            --subject-name "aio-opc-opcuabroker"
```

Comment:
There should be an example with explanation for explicit subject name and app uri dynamic